### PR TITLE
fixed dispatcher() on cutoff conditions

### PR DESCRIPTION
**Pull request by kristinaspring**
fixes https://github.com/xmidt-org/caduceus/issues/208

### CODE REVIEW

**Comment by schmidtw** [comment]
Remove the `Loop:` label.

**Comment by kristinaspring** [comment]
We need `Loop:` for this line though:
https://github.com/xmidt-org/caduceus/blob/b887423704e38e13c8f91a0d5d24af3d157e4020/outboundSender.go#L470

Otherwise, we would just `break` out of the select statement.  Is there a better way to do this?

File: outboundSender.go [hunk 1]
```diff
@@ -480,7 +480,7 @@ Loop:
 
 			if now.Before(dropUntil) {
 				obs.droppedCutoffCounter.Add(1.0)
-				break Loop
+				continue
```
Comment (johnabass) [inline outboundSender.go:483]: This works, but do we still need the `Loop` label at all?

Quick thought:  Seems like this code never exits now, which isn't much of an issue if we never intend to quiesce/restart it.

**Comment by gargidb** [comment]
I think we still need the `Loop` label to break out if the queue is empty.

**Pull request merged**